workshop/content/docs/setup/index.md

```diff
@@ -107,25 +105,24 @@ Next, you can override the default configuration and add your own data using *Do
 
 ### Custom Configuration
 
-Here we override the default configuration which resides at `/pygeoapi/local.config.yml` within the Container 
-with our local file [default.config.yml](https://github.com/geopython/pygeoapi/blob/master/docker/default.config.yml) 
-by using Docker Volume Mount (`-v` option). A Docker Volume mount attaches, 'mounts', a 
+In the upcoming exercises we are going to update the configuration file multiple times.
```

Comment:
Hmm, in reality there is already a pygeoapi config in the workshop materials (now under docker/pygeoapi/docker.config.yml) which is mounted via docker-compose and modified in the exercises.
Suggestion is to mount that file in this exercise here.

Also think we should have a Docker Compose with that file mounted in this section as to prepare for "first.md"

Reply:
+1

Reply:
it could also make sense to first run directly on a container, before going into the compose use case